log_injection/inyhow_ai-novel-generator/app.py

```
@@ -0,0 +1,238 @@
+            )
+        
+        # 记录成功响应
+        logging.info(f"成功生成内容: 标题='{title}', 章节数={len(chapters)}")
+            
+        return {
+            "success": True,
+            "title": title,
+            "chapters": chapters
+        }
+        
+    except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)